Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,606 @@
+    end
+    return
+  else
+    if self.EmptyStateText then
+      self.EmptyStateText:Hide()
+    end
+  end
+  
+  for portfolioId, portfolio in pairs(portfolios) do
+    local frame = self:CreatePortfolioFrame(portfolioId, portfolio)
+    if frame then
+      frame:SetPoint("TOPLEFT", self.Content, "TOPLEFT", 0, -yOffset)
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)